resources/views/admin/call_champions.blade.php

```diff
@@ -0,0 +1 @@
+call champ here
```

Comment:
these are views for other tabs i mean other than upload_data there are 3
more tabs right?

On Thu, Aug 11, 2016 at 8:15 AM, shashank-srikant <notifications@github.com>
wrote:

> In resources/views/admin/call_champions.blade.php
> <https://github.com/shashank-srikant/sevasetu-mothercare/pull/32#discussion_r74363282>
> :
>
> > @@ -0,0 +1 @@
> > +call champ here
>
> @Shubham2801 <https://github.com/Shubham2801>
> not sure what these changes are.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/shashank-srikant/sevasetu-mothercare/pull/32/files/7961619937a7148eb13a562f5b9f2d4b4ddeda3a#r74363282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AT4Bkhtm9e_G1VtA_og5k5SFVPtBRWUzks5qeozfgaJpZM4JeUVt>
> .
>